psi4/src/psi4/libmints/orthog.cc

```diff
@@ -40,18 +41,58 @@
 
 namespace psi {
 
-OverlapOrthog::OverlapOrthog(OrthogMethod method, SharedMatrix overlap, double lindep_tolerance, int print)
-    : orthog_method_(method), overlap_(overlap), lindep_tol_(lindep_tolerance), print_(print) {
+OverlapOrthog::OverlapOrthog(OrthogMethod method, SharedMatrix overlap, SharedVector rsq, double lindep_tolerance,
```

Comment:
Can we write this out as `OverlapOrthogonalization`? Not too much longer and messy it more clean (IMO).

Reply:
I hope @jturney agrees :)